<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1069</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1069</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionSmall"/>
    <addaction name="actionMedium"/>
    <addaction name="actionLarge"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="Small">
   <property name="text">
    <string>fasfa</string>
   </property>
  </action>
  <action name="actionSmall">
   <property name="text">
    <string>Маленький</string>
   </property>
  </action>
  <action name="actionMedium">
   <property name="text">
    <string>Средний</string>
   </property>
  </action>
  <action name="actionLarge">
   <property name="text">
    <string>Большой</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
